### PR TITLE
Fix Compatibility issue with Django 3.0 due to django.utils.six

### LINKED ISSUE #524: Compatibility issue with Django 3.0 due to django.utils.six
Django 3.0 scheduled for December removed Python 2 compatibility API used by easy-thumbnails (django.utils.six).

See https://docs.djangoproject.com/en/dev/releases/3.0/#removed-private-python-2-compatibility-apis
It would be wise to switch now to the direct use of six.

### PR DESCRIPTION
**Pull request by Mogost**
Fix #524

### CODE REVIEW

**Comment by davmlaw** [comment]
You forgot a few:
```
easy-thumbnails$ find . -name "*.py" | xargs grep django.utils.*six
./easy_thumbnails/alias.py:from django.utils import six
./easy_thumbnails/utils.py:from django.utils import six
./easy_thumbnails/options.py:from django.utils import six
./easy_thumbnails/templatetags/thumbnail.py:from django.utils import six
./easy_thumbnails/tests/test_files.py:from django.utils import six
./easy_thumbnails/tests/test_source_generators.py:from django.utils import six
./easy_thumbnails/processors.py:from django.utils import six
./easy_thumbnails/files.py:from django.utils import six
```

File: setup.py [hunk 1]
```diff
@@ -53,7 +53,8 @@ def read_files(*filenames):
         'django>=1.8;python_version>="3"',
         'six',
         'pillow<4;python_version<"2.7"',
-        'pillow;python_version>="2.7"',
+        'pillow<7;python_version<="2.7"',
+        'pillow;python_version>="3"',
```
Comment (jaap3) [inline setup.py]: I think you can drop the `'pillow<4;python_version<"2.7"'` bit as easy-thumbnails doesn't claim to support Python < 2.7.

```
'pillow<7;python_version<"3"',
'pillow;python_version>="3"',

File: tox.ini [hunk 2]
```diff
@@ -27,6 +29,7 @@ deps =
     django20: Django>=2.0,<2.1
     django21: Django>=2.1,<2.2
     django22: Django>=2.2a1,<3.0
+    django30: Django>=3.0,<3.1
     six
```
Comment (jaap3) [inline tox.ini]: I don't think six needs to be listed here now it's a dependency of easy-thumbnails

File: tox.ini [hunk 3]
```diff
@@ -35,6 +39,7 @@ commands =
 basepython = python3.6
 deps =
     Django
+    six
```
Comment (jaap3) [inline tox.ini]: Same here

**Comment by jrief** [comment]
Please drop support for Python-2.7 and do not add an external dependency to `six`.
Python-2 is dead and will not be supported anymore in 2020.

**Pull request closed without merging**